region/maridia/outer.json

```diff
@@ -1994,6 +2074,40 @@
                   ],
                   "note": "The second jump is doable by combining a stationary mid-air mockball with a mid-air springball jump.",
                   "devNote": "The second jump is actually doable with just a regular SpringBall jump but is ridiculously tight."
+                },
+                {
+                  "name": "Suitless Water Shinecharge",
+                  "notable": false,
+                  "requires": [
+                    "canWaterShineCharge",
+                    {"or": [
+                      {"canShineCharge": {
+                        "usedTiles": 33,
+                        "shinesparkFrames": 43,
+                        "excessShinesparkFrames": 26,
```

Comment:
We should add another Nodewith strats to deal with the pirate.

Reply:
the shinespark strat often avoids or kills the pirate, but it's a complex relationship. While doing a midair shinespark for example, sparking immediately at the peak of your jump, there is a way to kill or completely avoid the pirate when starting with 42-45 (sparking up left), 51, or 58+ energy. In general though, with low energy it is probably best to spark up left and careful jump past the pirate.

Reply:
There's some risky business going on here.  
Sparking to 6 avoids `stationaryLateralMidAirMorph` with springball, and lets you freeze the fish to 5 without jump assist.
Springball can jump around however so this is good for that; maybe a little scary around the pirate at 29 health.
To reach the top of 6 you need `crouchjumpdowngrab` and to pass through the pirate, when you're at 29 health.  From there you could freeze the fish, to save a few shinespark frames reaching 5.

The vertical spark is in the same situation.  Just needs to also not kill the fish.